### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by taxmeifyoucan**
Updated article, added links to new pages, updated database size overview numbers

### CODE REVIEW

File: src/content/developers/docs/nodes-and-clients/index.md [hunk 1]
```diff
@@ -72,6 +72,8 @@ If you run a full node, the whole Ethereum network benefits from it.
 
 ## Running your own node {#running-your-own-node}
 
+For general overview on running own Ethereum client instance, refer to [Spin up your own node article](/en/developers/docs/spin-up-your-node/).
```
Comment (ryancreatescopy) [inline src/content/developers/docs/nodes-and-clients/index.md:75]: Interested in running your own Ethereum client? Learn how to [spin up your own node](/en/developers/docs/spin-up-your-node/).

File: src/content/developers/docs/nodes-and-clients/index.md [hunk 2]
```diff
@@ -93,7 +95,7 @@ If you run a full node, the whole Ethereum network benefits from it.
 
 ## Alternatives {#alternatives}
 
-Running your own node can be difficult and you don’t always need to run your own instance. In this case, you can use a third party API provider like [Infura](https://infura.io), [Alchemy](https://alchemyapi.io), or [QuikNode](https://www.quiknode.io). Alternatively [ArchiveNode](https://archivenode.io/) is a community-funded Archive node that hopes to bring archive data on the Ethereum blockchain to independent developers who otherwise couldn't afford it.
+Running your own node can be difficult and you don’t always need to run your own instance. In this case, you can use a third party API provider like [Infura](https://infura.io), [Alchemy](https://alchemyapi.io), or [QuikNode](https://www.quiknode.io). Alternatively [ArchiveNode](https://archivenode.io/) is a community-funded Archive node that hopes to bring archive data on the Ethereum blockchain to independent developers who otherwise couldn't afford it. For overview of using these services, refer to [Nodes as a services article](/en/developers/docs/nodes-and-clients/nodes-as-a-service/).
```
Comment (ryancreatescopy) [inline src/content/developers/docs/nodes-and-clients/index.md:98]: We don't need to label the links as an article. 

For an overview of using these services, check out [nodes as a services](/en/developers/docs/nodes-and-clients/nodes-as-a-service/)

**Review by ryancreatescopy — CHANGES_REQUESTED** [review]
Couple of content tweaks, but thanks for updating @taxmeifyoucan !

**Comment by taxmeifyoucan** [comment]
@ryancreatescopy thanks for reviewing! I tweaked the content and updated pictures with database size charts, changed Parity to Openethereum.

File: src/content/developers/docs/nodes-and-clients/index.md [hunk 3]
```diff
@@ -193,16 +195,16 @@ Depending on which software and sync mode are you going to use, hundreds of GBs
...
+| OpenEthereum | 280GB+                | 6TB+                   |
+| Nethermind   | 200GB+                | 5TB+                     |
+| Besu         | 750GB+                | 5TB+                     |
 
 ![A chart showing that GB needed for a full sync is trending up](./full-sync.png)
 
 ![A chart showing that GB needed for an archive sync is trending up](./archive-sync.png)
 
-These charts show how storage requirements are always changing. For the most up-to-date data for Geth and Parity, see the [full sync data](https://etherscan.io/chartsync/chaindefault) and [archive sync data](https://etherscan.io/chartsync/chainarchive).
+These charts show how storage requirements are always changing. For the most up-to-date data for Geth and Openethereum, see the [full sync data](https://etherscan.io/chartsync/chaindefault) and [archive sync data](https://etherscan.io/chartsync/chainarchive).
```
Comment (wackerow) [inline src/content/developers/docs/nodes-and-clients/index.md:207]: "Openethereum" -> "OpenEthereum"

**Review by wackerow — CHANGES_REQUESTED** [review]
@taxmeifyoucan Just one small correction to "OpenEthereum", otherwise looking good! 👍 thanks for this!

File: src/content/developers/docs/nodes-and-clients/index.md (hunk 3 above)
Comment (samajammin) [inline src/content/developers/docs/nodes-and-clients/index.md:207]:
```suggestion
These charts show how storage requirements are always changing. For the most up-to-date data for Geth and OpenEthereum, see the [full sync data](https://etherscan.io/chartsync/chaindefault) and [archive sync data](https://etherscan.io/chartsync/chainarchive). 
```

**Review by samajammin — APPROVED** [review]
Thanks again!

**Pull request merged**